### DISCUSSION TITLE
have textbooks for describing different species advantages and disadvantages

### DISCUSSION THREAD

**Discussion by Bod9001** [unitystation/unitystation#7085] [Ideas]
also have restrictions on  what can be implanted into what, like only allowing 4 appendages on torso

**Comment by NoooneyDude** [comment]
Relevant conversation [here](https://discord.com/channels/273774715741667329/312454684021620736/865212194321072128).

**Comment by corp-0** [comment]
I am making this a discussion so we can use it to discuss the advantages and the presentation.
